packages/trader/src/Stores/Modules/Trading/Helpers/active-symbols.js

```diff
@@ -37,19 +37,23 @@ export const pickDefaultSymbol = (active_symbols = []) => {
 };
 
 const getFavoriteOpenSymbol = (active_symbols) => {
-    const chart_favorites = LocalStore.get('cq-favorites');
-    if (!chart_favorites) return undefined;
-    const client_favorite_markets = JSON.parse(chart_favorites)['chartTitle&Comparison'];
+    try {
+        const chart_favorites = LocalStore.get('cq-favorites');
+        if (!chart_favorites) return undefined;
+        const client_favorite_markets = JSON.parse(chart_favorites)['chartTitle&Comparison'];
 
-    const client_favorite_list = client_favorite_markets
-        .map(client_fav_symbol => active_symbols
-            .find(symbol_info => symbol_info.symbol === client_fav_symbol));
-    if (client_favorite_list) {
-        const client_first_open_symbol = client_favorite_list
-            .filter(symbol => symbol).find(isSymbolOpen);
-        if (client_first_open_symbol) return client_first_open_symbol.symbol;
+        const client_favorite_list = client_favorite_markets
+            .map(client_fav_symbol => active_symbols
+                .find(symbol_info => symbol_info.symbol === client_fav_symbol));
+        if (client_favorite_list) {
+            const client_first_open_symbol = client_favorite_list
+                .filter(symbol => symbol).find(isSymbolOpen);
+            if (client_first_open_symbol) return client_first_open_symbol.symbol;
+        }
+        return undefined;
+    } catch (error) {
+        return undefined;
```

Comment:
Can you confirm which of these two files are being used? maybe we should let go of the other one to be consistent.

Reply:
the one with trade, but since I don't know which one will used in future, I edited both

Reply:
@reza-fs I believe this helper is being used in both places, but we can work on refactoring it in another housekeeping PR.